src/lib/containers/entity/annotations/AnnotationEditorUtils.ts

```diff
@@ -42,56 +46,60 @@ export function getFriendlyPropertyName(error: AjvError) {
   }
 }
 
-export function transformErrors(errors: AjvError[]): AjvError[] {
```

Comment:
`transformErrors` needs the concreteType of the entity now. The concreteType cannot be added to the function signature because this function is passed to RJSF.

So, wrap the original function in a closure to capture the concrete type.